src/test/java/edu/usf/cutr/gtfsrtvalidator/FeedMessageTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.usf.cutr.gtfsrtvalidator;
+
+import com.google.transit.realtime.GtfsRealtime;
+import edu.usf.cutr.gtfsrtvalidator.helper.ErrorListHelperModel;
+import edu.usf.cutr.gtfsrtvalidator.test.headerandentity.FeedEntityTest;
+import edu.usf.cutr.gtfsrtvalidator.test.headerandentity.FeedHeaderTest;
+import java.io.File;
+import java.io.IOException;
+import java.util.Date;
+import junit.framework.TestCase;
+import org.onebusaway.gtfs.impl.GtfsDaoImpl;
+import org.onebusaway.gtfs.serialization.GtfsReader;
+import org.onebusway.gtfs_realtime.exporter.GtfsRealtimeLibrary;
+
+
+public class FeedMessageTest extends TestCase{
+    
+    public GtfsDaoImpl gtfsData, gtfsData2; //gtfsData2 contains location_type = 1 for stop_id
+    public GtfsReader reader;
+    public final File staticGtfs = new File("src/test/resources/testagency.zip");
+    public final File staticGtfs2 = new File("src/test/resources/testagency2.zip");
+    public final static String ENTITY_ID = "TEST_ENTITY";
+    public final static long POSIX_TIME_PASS_VALIDATION = new Date().getTime()/1000; // current date in seconds
+    public final static long POSIX_TIME_FAIL_VALIDATION = 12345;
```

Comment:
The most frequent occurrence I've seen of POSIX errors is failing to divide by 1000 (i.e., using milliseconds instead of seconds past epoch), so I'd prefer if this was `public final static long POSIX_TIME_FAIL_VALIDATION = new Date().getTime();` to imitate that case.